### PR TITLE
added react components

### PR DESCRIPTION
**Pull request by ishpaul777**
# In this Pull Request I implemented the project requirements -

- Initialise React app.
- The building blocks of your app should be set as re-usable components.
- Added [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>s `and `<Link>s` for the app's navigation:
> - Books - the default view:
       - to display the list of books (empty at this point but it should be ready for the data) with the Remove button (no 
       functionality yet).
       - Creates a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a 
       component that displays a list of books.
       - Should have a form for adding a book (no functionality yet).
> - Create a separate component for this form (with inputs for a title and an author).
Categories:
To display  "Under Construction..." only.

### CODE REVIEW

File: src/Components/CategoriesContainer.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const CategoriesContainer = () => (
+  <div className="categoriesContainer">
+    <Navbar />
+    <h2>Under Construction...</h2>
```
Comment (topeogunleye) [inline src/Components/CategoriesContainer.js]: - [x] As written in the project requirement. I kindly suggest that your categories page should display only a button with text "Check status" for now.

image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ishpaul777, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
- [x] I kindly suggest that you personalize your README file content and put your project information there because currently only the placeholders from Microverse Template are there as you haven't made any changes to the template.
- [x]  Next time, and for future commits, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image
### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- Nothing to mention.

- [x] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,56 +1,39 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
 
+ > Book Store is a website for all Book Reader. It is a Single Page App (SPA) that allows users to:
```
Comment (Deepakdanger) [inline README.md:2]: - [ ] [OPTIONAL] Kindly add one proper header in the readme file.

**Review by Deepakdanger — APPROVED** [review]
Hi @ishpaul777, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- _Check the comment below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**